src/io/IOHelpers.h

```diff
@@ -8,14 +8,49 @@ class DomainDecompBase;
 
 namespace IOHelpers {
 
+/**
+ * Removes the momentum from the simulation.
+ * Will first sum up the overall momentum of the simulation and then apply a shift to all particles, s.t., there is no
+ * more total momentum.
```

Comment:
After reading this I know exactly as much as before :D 
Describe how it achieves this (subtracting something -explain what something is- from the velocity) or at least that it has the side effect of changing the velocity of all particles.